ios/NativeViews/IpfsImageViewManager.h

```diff
@@ -0,0 +1,5 @@
+#import <React/RCTViewManager.h>
+
+@interface IpfsImageViewManager : RCTViewManager
```

Comment:
This manager class is just standard RN native UI component stuff... You have to create the actual view implementation plus a 'manager' class that provides the RN runtime with the metadata it needs to be able to register and use your component.